ClientApp/viking-vault/src/UserDashboard/ViewAutomaticPayments.css

```diff
@@ -11,4 +11,24 @@
 .automatic-payments-list {
     position: relative;
     right: 4%;
-}
\ No newline at end of file
+}
+
+.add_payment_icon {
```

Comment:
Keep the class naming convention consistent, your application uses `kebab-case` for CSS naming, use that here as well